libr/anal/class.c

```diff
@@ -0,0 +1,1129 @@
+/* radare - LGPL - Copyright 2018 - thestr4ng3r */
+
+#include <r_anal.h>
+#include <r_vector.h>
+#include "../include/r_anal.h"
+
+#define CLASSES_FLAGSPACE "classes"
+
+static char *sanitize_id(const char *id) {
```

Comment:
Maybe move this function in `libr/util/string.c` instead?

Reply:
Perhaps something like `r_str_sanitize_sdb()` to sanitize specifically for usage in sdb keys?